examples/basic/default/Makefile

```diff
@@ -27,7 +27,10 @@ DEVELHELP ?= 1
 # Change this to 0 show compiler invocation lines by default:
 QUIET ?= 1
 
+CFLAGS += "-DCONFIG_NANOCOAP_BLOCK_SIZE_EXP_MAX=10"
+
 # Modules to include:
+USEMODULE += slipdev_stdio
```

Comment:
I would prefer this to be its own example.

Reply:
> The second commit adds an usage example, where the shell commands are exposed to coap and through it to slipmux. That commit will be dropped prio merging.

Thanks for the review!

Reply:
I resolved your other comments without fixing them as they would be dropped too.